src/PROPOSAL/PROPOSAL/math/Spherical3D.h

```diff
@@ -34,4 +34,22 @@ namespace PROPOSAL  {
     protected:
         void print(std::ostream&) const override;
      };
+
+
+    /*!
+     * Container for a spherical vector with radius 1
+     *
+     * \param zenith
+     * \param azimuth
+     */
+     struct UnitSphericalVector{
+        double zenith;
+        double azimuth;
+    
+        UnitSphericalVector() = default;
```

Comment:
shouldn't this also be constexpr?

Reply:
hmm, I tried but then I get the error
```defaulted definition of default constructor is not constexpr```

Reply:
ah, they need default values
but should they have default values?